Week-3/Session-09/JavaScript number guessing game/script.js

```diff
@@ -0,0 +1,35 @@
+// Your solution goes here
+const playGuessingGame = (numToGuess, totalGuesses = 10) => {
+	let guessCount = 0;
+	let guess = Number(prompt(`Enter a number between 1 and 100.`));
+
+	if (guess === null) {
```

Comment:
This pattern of nested if-else really bad, I will tell you how to write in our one on one.